cross-media-measurement/src/main/cc/wfa/measurement/common/math/distributions.cc

```diff
@@ -77,4 +77,13 @@ absl::StatusOr<int64_t> GetDistributedGeometricRandomComponent(
   return options.shift_offset + polya_a - polya_b;
 }
 
+absl::StatusOr<int64_t> GetTruncatedDiscreteLaplaceDistributedRandomNumber(
```

Comment:
This is not quite how tDlap is defined in the paper. Specifically, if we want to sample tDlap as defined, we would keep sampling from (centered) tDlap until it falls in the desired range [-mu, mu], and then output that plus mu.

I think the approach you're using here also works, but might lose a factor of 2 in the delta.

Reply:
Done, I knew the difference, but thought it was subtle and was too lazy to implement the actual algorithm. Thanks for pointing this out.

Reply:
I see... Yeah this new code LGTM. (I think it's a good idea to be consistent with the write-up as much as we can.)